src/SUMMARY.md

```diff
@@ -93,4 +93,9 @@
 [Appendix C: Glossary](./appendix/glossary.md)
 [Appendix D: Code Index](./appendix/code-index.md)
 [Appendix E: Bibliography](./appendix/bibliography.md)
+
+[Appendix Z HumorRust](./appendix/humorust.md)
```

Comment:
Looks like there should have been a `:` after the `Z`.

Reply:
@Phrohdoh Yep, this was fixed recently.